.env.example

```
@@ -1 +1 @@
-VITE_MAPTILER_API_KEY=
\ No newline at end of file
+VITE_MAPTILER_API_KEY=#Your Maptiler key here#
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)